<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="AcceptAnswer">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>170</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Accept Answer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ExitBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OpenFileBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Open file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="NextRoundBtn">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>220</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Next Round</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StartAgainBtn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>230</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Answer">
    <property name="geometry">
     <rect>
      <x>172</x>
      <y>80</y>
      <width>461</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>461</width>
      <height>60</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLabel" name="Question">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>45</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="log">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>300</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Log:</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>108</width>
      <height>72</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QProgressBar" name="All">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 85, 255);</string>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="format">
      <string>%v</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="Good">
     <property name="value">
      <number>0</number>
     </property>
     <property name="format">
      <string>%v</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="Bad">
     <property name="value">
      <number>0</number>
     </property>
     <property name="format">
      <string>%v</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
